src/transformers/models/auto/configuration_auto.py

```diff
@@ -485,6 +486,7 @@
         ("mask2former", "Mask2Former"),
         ("maskformer", "MaskFormer"),
         ("maskformer-swin", "MaskFormerSwin"),
+        ("matcha", "Pix2Struct"),
```

Comment:
Just to confirm, when adding those models in other mappings (auto processor, etc), I hit: 
```
`deplot` appears in the mapping `PROCESSOR_MAPPING_NAMES` but it is not defined in the keys of `CONFIG_MAPPING_NAMES`.
```
I think we shouldn't add these models in other mappings no?

Reply:
It should be in `CONFIG_MAPPING_NAMES` that is what I'm saying. And then in the other ones too so that the default model/processor are known.

Reply:
Maybe I am missing something, but when adding them on `CONFIG_MAPPING_NAMES`, I hit: 
```bash
  File "<frozen importlib._bootstrap>", line 1007, in _find_and_load
  File "<frozen importlib._bootstrap>", line 984, in _find_and_load_unlocked
ModuleNotFoundError: No module named 'transformers.models.deplot'
```

when running make fixup, probably because:
```python
self._modules[module_name] = importlib.import_module(f".{module_name}", "transformers.models")
```
[in `configuration_auto.py`](https://github.com/huggingface/transformers/blob/48fbd8fa2ebfcf35c14f3bca4055631bf83e7e8b/src/transformers/models/auto/configuration_auto.py#L639). Let me know if I misunderstood anything!

Reply:
Oh it's  already in `MODEL_MAPPING_NAMES` sorry about that. It should be there with the right names though, that's what got me confused.

Reply:
Ok thanks for clarifying!